changelog/Microsoft.Partner.CustomerSvcAdmin.json

```
@@ -0,0 +1,166 @@
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Resource",
+          "ChangedApiName": "delegatedAdminCustomer",
+          "ChangeType": "Addition",
+          "Description": "Added the [delegatedAdminCustomer](https://docs.microsoft.com/en-us/graph/api/resources/delegatedAdminCustomer?view=graph-rest-beta) resource",
+          "Target": "delegatedAdminCustomer"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Resource",
+          "ChangedApiName": "delegatedAdminRelationship",
+          "ChangeType": "Addition",
+          "Description": "Added the [delegatedAdminRelationship](https://docs.microsoft.com/en-us/graph/api/resources/delegatedAdminRelationship?view=graph-rest-beta) resource",
+          "Target": "delegatedAdminRelationship"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Resource",
+          "ChangedApiName": "delegatedAdminRelationshipOperation",
+          "ChangeType": "Addition",
+          "Description": "Added the [delegatedAdminRelationshipOperation](https://docs.microsoft.com/en-us/graph/api/resources/delegatedAdminRelationshipOperation?view=graph-rest-beta) resource",
+          "Target": "delegatedAdminRelationshipOperation"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Resource",
+          "ChangedApiName": "delegatedAdminRelationshipRequest",
+          "ChangeType": "Addition",
+          "Description": "Added the [delegatedAdminRelationshipRequest](https://docs.microsoft.com/en-us/graph/api/resources/delegatedAdminRelationshipRequest?view=graph-rest-beta) resource",
+          "Target": "delegatedAdminRelationshipRequest"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Resource",
+          "ChangedApiName": "delegatedAdminServiceManagementDetail",
+          "ChangeType": "Addition",
+          "Description": "Added the [delegatedAdminServiceManagementDetail](https://docs.microsoft.com/en-us/graph/api/resources/delegatedAdminServiceManagementDetail?view=graph-rest-beta) resource",
+          "Target": "delegatedAdminServiceManagementDetail"
+        },
+        {
+          "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+          "ApiChange": "Resource",
+          "ChangedApiName": "tenantRelationship",
+          "ChangeType": "Addition",
+          "Description": "Added the [tenantRelationship](https://docs.microsoft.com/en-us/graph/api/resources/tenantRelationship?view=graph-rest-beta) resource",
+          "Target": "tenantRelationship"
+        }
+      ],
+      "Id": "864c77db-f273-4a11-8104-8149f3f9566f",
+      "Cloud": "Prod",
+      "Version": "beta",
+      "CreatedDateTime": "2022-02-16T23:33:05.8733913Z",
+      "WorkloadArea": "Microsoft.Partner.CustomerSvcAdmin",
+      "SubArea": ""
```

```suggestion
      "WorkloadArea": "Microsoft.Partner.CustomerSvcAdmin",
      "SubArea": ""
```
Let's use these values as per the internal thread. This API seems to be about directory management only in a partner-customer r/ship
```suggestion
      "WorkloadArea": "Identity and access",
      "SubArea": "Directory management"
```